fbw-a380x/src/systems/instruments/src/EWD/elements/WdLimitations.tsx

```diff
@@ -52,8 +52,11 @@ export class WdLimitations extends DisplayComponent<WdLimitationsProps> {
         .join('\r'),
     );
 
-    this.limitationsLeftSvgRef.instance.style.height = `${(this.limitationsLeftSvgRef.instance.getBBox().height + 12).toFixed(1)}px`;
-    this.limitationsRightSvgRef.instance.style.height = `${(this.limitationsRightSvgRef.instance.getBBox().height + 12).toFixed(1)}px`;
+    // Weirdly enough, we have to wait for the SVG to be rendered to get its bounding box
```

Comment:
Can you try putting the code in the „onAfterRender“ method and see if it solves the issue?

Reply:
Wouldn’t solve it, since this has to be called every time after the content of the SVG changes, hence the sub. 
a better solution would be to have this run after a callback notifies of the finished layouting of the SVG, but I haven’t found that so far :/

Reply:
Understood, we can improve that later then if we find a better solution 👍

Reply:
Sounds good 👍 Or even better, the next version won’t require SVGs but just use the existing flex box which does the resizing automatically